src/Lykke.Service.BlockchainCashoutPreconditionsCheck.Core/Domain/Validation/ValidationError.cs

```diff
@@ -13,5 +13,13 @@ public static ValidationError Create(ValidationErrorType type, string value)
                 Value = value
             };
         }
+
+        public static ValidationError Create(string value)
+        {
+            return new ValidationError
+            {
+                Value = value
```

Comment:
Please, add
`Type = ValidationErrorType.Error;`
otherwise, the `Type` prop will be filled by the default value for `(int)ValidationErrorType`, which is `0` and will be associated with `ValidationErrorType.None`. In turn, this may lead to wrong responce interpretation by a caller (let's assume he checks only `Type` from the validation result he gets).